source/ed/connectors/gtfs_parser.h

```diff
@@ -214,20 +216,23 @@ struct DefaultContributorHandler : public GenericHandler {
 };
 
 struct StopsGtfsHandler : public GenericHandler {
-    StopsGtfsHandler(GtfsData& gdata, CsvReader& reader) : GenericHandler(gdata, reader) {}
+    StopsGtfsHandler(GtfsData& gdata, CsvReader& reader) : GenericHandler(gdata, reader), stop_code_is_present(false) {}
     using stop_point_and_area = std::pair<ed::types::StopPoint*, ed::types::StopArea*>;
 
     int id_c, code_c, name_c, desc_c, lat_c, lon_c, zone_c, type_c, parent_c, wheelchair_c, platform_c, timezone_c,
         ext_code_c;
 
     int ignored = 0;
+    bool stop_code_is_present;
```

Comment:
Minor - if you use ` bool stop_code_is_present = false;` that would save you from initialising it in the constructor.